src/codewhisperer/commands/startSecurityScan.ts

```diff
@@ -78,6 +79,9 @@ export async function startSecurityScan(
             throw new Error(`"${editor.document.languageId}" is not supported for security scan.`)
         }
         const uri = dependencyGraph.getRootFile(editor)
+        const projectSize = await dependencyGraph.getProjectSize(uri)
```

Comment:
If this `projectSize` is only used in telemetry reporting, is it possible to let `await dependencyGraph.getProjectSize(uri)` not block the execution of `startSecurityScan` logic?

Reply:
Updated